server/src/main/java/org/opensearch/index/shard/IndexingStats.java

```diff
@@ -194,7 +199,8 @@ public Stats(
             long noopUpdateCount,
             boolean isThrottled,
             long throttleTimeInMillis,
-            DocStatusStats docStatusStats
+            DocStatusStats docStatusStats,
+            long maxLastIndexRequestTimestamp
```

Comment:
Thanks for sticking with this @rgsriram. If you run `./gradlew japicmp` it will produce a file at `server/build/reports/java-compatibility/report.html` that will show you breaking changes. In this case it is the change to this constructor. Unfortunately this class is exposed through the plugin API and we try to keep signatures stable within a major version. To get around this, can you keep the existing constructor that calls the new constructor with a default value for the new field?